sections/futures/Transfers/Transfers.tsx

```diff
@@ -0,0 +1,183 @@
+					{
+						Header: (
+							<StyledTableHeader>
+								{t('futures.market.user.transfers.table.action')}
+							</StyledTableHeader>
+						),
+						accessor: 'action',
+						Cell: (cellProps: any) => <StyledActionCell>{cellProps.value}</StyledActionCell>,
+						width: 50,
+					},
+					{
+						Header: (
+							<StyledTableHeader>
+								{t('futures.market.user.transfers.table.amount')}
+							</StyledTableHeader>
+						),
+						accessor: 'amount',
+						sortType: 'basic',
+						Cell: (cellProps: any) => (
+							<StyledAmountCell isPositive={cellProps.row.original.isPositive}>
+								{cellProps.value}
+							</StyledAmountCell>
+						),
+						sortable: true,
+						width: 50,
+					},
+					{
+						Header: (
+							<StyledTableHeader>{t('futures.market.user.transfers.table.date')}</StyledTableHeader>
+						),
+						accessor: 'timestamp',
+						Cell: (cellProps: any) => (
+							<DefaultCell>{timePresentation(cellProps.value)}</DefaultCell>
+						),
+						width: 50,
+					},
+					{
+						Header: (
+							<StyledTableHeader>
+								{t('futures.market.user.transfers.table.transaction')}
+							</StyledTableHeader>
+						),
+						accessor: 'txHash',
+						Cell: (cellProps: any) => {
+							return (
+								<DefaultCell>
+									<StyledExternalLink href={blockExplorerInstance?.txLink(cellProps.value)}>
+										{truncateAddress(cellProps.value)}
+									</StyledExternalLink>
+								</DefaultCell>
+							);
+						},
+						width: 50,
+					},
+				]}
+				data={marginTransfers}
+				columnsDeps={columnsDeps}
+				isLoading={isLoading && !isLoaded}
+				noResultsMessage={
+					isLoaded && marginTransfers?.length === 0 ? (
+						<TableNoResults>
+							<StyledTitle>{t('futures.market.user.transfers.table.no-results')}</StyledTitle>
+						</TableNoResults>
+					) : undefined
+				}
+				showPagination={true}
+			/>
+		</TableContainer>
+	);
+};
+
+export default Transfers;
+
+const StyledTable = styled(Table)`
+	margin-top: 16px;
+`;
+
+const DefaultCell = styled.p`
+	color: ${(props) => props.theme.colors.common.primaryWhite};
+`;
+
+const TableContainer = styled.div`
+	margin-bottom: 15px;
+`;
+
+const StyledActionCell = styled(DefaultCell)`
+	text-transform: capitalize;
+`;
+
+const StyledTitle = styled.p`
+	color: ${(props) => props.theme.colors.common.primaryWhite};
+	font-size: 16px;
+	margin: 0;
+`;
+
+const StyledExternalLink = styled(ExternalLink)`
+	color: ${(props) => props.theme.colors.common.primaryWhite};
+	text-decoration: underline;
+	&:hover {
+		text-decoration: underline;
+	}
+`;
+
+const StyledAmountCell = styled(DefaultCell)<{ isPositive: boolean }>`
+	color: ${(props: any) =>
+		props.isPositive
+			? props.theme.colors.common.primaryGreen
+			: props.theme.colors.common.primaryRed};
+`;
+
+const StyledTableHeader = styled.div`
+	font-family: ${(props) => props.theme.fonts.bold};
+	text-transform: capitalize;
+`;
+
+const TableNoResults = styled(GridDivCenteredRow)`
```

Comment:
I think we need to improve this CSS. The borders don't exactly line up the way a regular table row does. The font size/type also looks out of place.

<img width="1110" alt="image" src="https://user-images.githubusercontent.com/10401554/165604587-b507bcb4-394e-4e6d-8b71-fbaff540781c.png">

Reply:
yeah i thought i addressed this, my bad - fix incoming